src/panels/FileListing/index.js

```diff
@@ -77,6 +91,11 @@ const FileListing = React.createClass({
         <a href={`api/v1/item/${file._id}/download`} target="_blank">
           <i className={style.downloadIcon}></i>
         </a>
+        {/* 500,000B (500KB) maximum preview-able size */}
```

Comment:
JSX commment
<https://facebook.github.io/react/docs/jsx-in-depth.html#comments>

On Thu, May 12, 2016 at 11:57 AM, Chris Harris <notifications@github.com>
wrote:

> In src/panels/FileListing/index.js
> <https://github.com/Kitware/HPCCloud/pull/390#discussion_r63066747>:
>
> > @@ -77,6 +91,11 @@ const FileListing = React.createClass({
> >          <a href={`api/v1/item/${file._id}/download`} target="_blank">
> >            <i className={style.downloadIcon}></i>
> >          </a>
> > +        {/* 500,000B (500KB) maximum preview-able size */}
>
> Just out of interest whats with the braces around the comment?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Kitware/HPCCloud/pull/390/files/5f6dad97c024877f483c2a5b24c4c30ae899f2e0#r63066747>
>



-- 
- Tristan Wright

*R&D Engineer*
*Kitware Inc.*